### PR TITLE
Anybar: export as default function

### PR DESCRIPTION
**Pull request by krenor**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [source](https://github.com/sindresorhus/anybar/blob/master/index.js#L5)
- [x] Increase the version number in the header if appropriate.
- ~~[ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`~~.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: types/anybar/index.d.ts [hunk 1]
```diff
@@ -5,6 +5,4 @@
 
 declare namespace anybar { }
 
-declare function anybar(color: string, options?: { port: number }): void
-
-export = anybar
+export default function anybar(color: string, options?: { port: number }): void
```
Comment (DanielRosenwasser) [inline types/anybar/index.d.ts]: This isn't correct, [it's definitely an `export =` declaration](https://github.com/sindresorhus/anybar/blob/8371af96bcb6ff3dee5af6b4b4f02153f4b84eb7/index.js#L5) and should be imported with `import anybar = require("anybar")` or as `import anybar from 'anybar'` by users who want to use 2.7's `esModuleInterop` or 1.8's `allowSyntheticDefaultImports`. [Read more here](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/df0a8726871810b6bab0ae6cc1ce5d1ea58b827d#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Could you instead remove the empty `declare namespace anybar {}`, fix up the version to `2.0` and enable whatever lint rules are disabled?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/anybar/index.d.ts (hunk 1 above)
Comment (krenor) [reply types/anybar/index.d.ts]: Why does `export function anybar` not work while using the  `declare function anybar ... export = anybar` approach it does? The first one results in `Cannot invoke an expression whose type lacks a call signature`. I'm not experienced enough to understand why, maybe you can clear this up for me? :)

Comment (DanielRosenwasser) [reply types/anybar/index.d.ts]: `export function anybar` gets emitted closer to something like

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**